configure

```diff
@@ -46,9 +48,9 @@ fi
 
 if [ $msg -ne 0 ]; then
   echo "*** Compilation will now be attempted and if it works you can ignore this message. In"
-  echo "*** particular, this should be the case on Mac where zlib is built in."
-  echo "*** However, if compilation fails, try 'locate zlib.h zconf.h' and ensure the zlib"
-  echo "*** development library is installed :"
+  echo "*** particular, this should be the case on Mac where zlib is built in or pkg-config"
+  echo "*** is not installed. However, if compilation fails, try 'locate zlib.h zconf.h' and"
+  echo "*** ensure the zlib development library is installed :"
   echo "***   deb: zlib1g-dev (Debian, Ubuntu, ...)"
```

Comment:
doesn't this turn off zlib support on mac by default

Reply:
If `pkg-config` is not available on Mac then yes, it turns off. Needs testing anyway.
There is a related comment above:
>  if pkg-config is not available then zlib will be disabled for higher chance of compilation success

Reply:
Note that branch was hit before when pkg-config was not installed, just this console message was not mentioning that.

Reply:
We could eventually try to check if zlib is available by attempting to compile dummy code, like we do for openmp. Docker image in your report didn't have either pkg-config and zlib. So then we could still compile with zlib support. For now I don't think it is worth. Messages are clear and points users in the proper direction. Installing pkg-config and/or zlib.